scheduler/docs/scheduler-rest-api.asc

```diff
@@ -75,6 +75,7 @@ The request body is list of job maps with each entry containing a map of the fol
 |`mem` | double | MB of requested memory.
 |`uris` | list of URI objects | A list of URIs that will be fetched into the container before launch.
 |`env` | JSON map | A map of environment variables to be provided to the job.
+|`host_constraints` | JSON map | A map of host constraints to be provided to the job.
```

Comment:
Once we cement the semantics, we'll need to make this more descriptive, and possibly include it in its own section. Also, we should indicate somewhere that it's optional.